flink-connectors/flink-connector-kafka/src/main/java/org/apache/flink/connector/kafka/source/reader/deserializer/KafkaValueOnlyDeserializerWrapper.java

```diff
@@ -36,10 +36,15 @@
 
 /** A package private class to wrap {@link Deserializer}. */
 class KafkaValueOnlyDeserializerWrapper<T> implements KafkaRecordDeserializationSchema<T> {
+
     private static final long serialVersionUID = 5409547407386004054L;
+
     private static final Logger LOG =
             LoggerFactory.getLogger(KafkaValueOnlyDeserializerWrapper.class);
+
     private final Class<? extends Deserializer<T>> deserializerClass;
+    // always be false since this Deserializer is only used for value.
```

Comment:
nits: The comment seems to be outdated since this value could be true.

And could we start the comment with an upper-case letter for consistency with other comments?

Reply:
It must always be false, since this class is only used for Kafka value. Otherwise the name this class should be changed.

Reply:
I see why its value is always false. Since its value is always false, would it be simpler to just use `deserializer.configure(config, false)`? We could add a comment above if needed.

Reply:
It is because of code style consistency and don't hide the magic deeply into the code at the corner.

Reply:
Hmm... do you mean we add `isKey` here to be consistent with `KafkaSerializerWrapper`? I am not sure we need to do this. Note that `KafkaSerializerWrapper`'s constructor takes `isKey` as parameter but `KafkaValueOnlyDeserializerWrapper` does not. So these two classes will anyway be inconsistent.

Maybe @fapaul can also comment on this?

Reply:
Hmm, I'd favor removing the instance variable and directly calling `deserializer.configure(config, false);` and move the comment above this line

Reply:
Code style consistency means in this case that developers, when the come from `KafkaSerializerWrapper`  will check the `isKey` at first to see if `deserializer.configure(...)` has been used in `KafkaValueOnlyDeserializerWrapper`. It is much easier to check it via instance variable than digging into the code. Try to read both classes back and forth, you will get the feeling.

It is not a big deal. I will change it back.

Reply:
Thank you Jing for digging into this.